src/qukeys/Qukeys.cpp

```diff
@@ -20,154 +20,197 @@ namespace qukeys {
 
 // Event handler
 EventHandlerResult Plugin::onKeyswitchEvent(KeyEvent& event) {
-  // If this plugin isn't active:
-  if (! plugin_active_) {
-    if (isQukeysKey(event.key)) {
-      Qukey qukey = getQukey(event.key);
-      event.key = qukey.primaryKey();
-    }
-    return EventHandlerResult::proceed;
-  }
-
-  // If the key toggled on
-  if (event.state.toggledOn()) {
+  // This function is a bit misleading. All it does is add the event to the
+  // queue and abort; the processing of the queue now happens in the pre-scan
+  // hook instead. I tried including tests here for events that could bypass the
+  // event queue, but it increases the size of the binary by more than seems
+  // worthwhile (66 bytes). Similarly, I had included a call to `processQueue()`
+  // after the event gets queued, but that happens in the pre-scan hook anyway,
+  // so it saves another 6 bytes of PROGMEM.
+  event_queue_.append(event);
```

Comment:
I need to call `processQueue()` here to guarantee that `event_queue_` can't overflow.